app/assets/stylesheets/components/buttons.scss

```diff
@@ -410,3 +410,91 @@
     }
   }
 }
+
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+// .c-btn //////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+
+.c-btn {
+  position: relative;
+  display: inline-block;
+  padding: var(--su-2) var(--su-4);
+  border-radius: var(--radius);
+  font: inherit;
+  text-align: center;
+  cursor: pointer;
+  transition: all var(--transition-props);
+  border: none;
+  outline: 0;
+  overflow-wrap: normal;
+
+  background-color: var(--bg);
+  color: var(--color);
+
+  &:hover:enabled,
+  &:focus-visible {
+    background-color: var(--bg-hover);
+    color: var(--color-hover);
+  }
+
+  &:focus-visible {
+    box-shadow: var(--focus-ring);
+  }
```

Comment:
`focus-visible` isn't supported in Safari yet... which means we'll need to use the js-focus-visible polyfill to make this work for us.

The alternative is to just use `:focus` which in Chrome/Firefox is already implemented as 'focus-visible' under the hood, but I think in some versions of Safari might result in the focus style being visible on click with mouse/touch device

Reply:
daaaaamn, i forgot about that... lets roll with polyfill since we already use it anyway..

Reply:
Yeah the `:focus-visible` polyfill appears to be only used in Storybook atm.

https://github.com/forem/forem/blob/89ac6f1b5a0ee6d8dbaaeffea573081041089e85/app/javascript/.storybook/preview.js#L5

we can load it in`packs/base.jsx`. If it looks like it affects performance, we can dynamically import it.

Reply:
TIL! yea, i think we should bring it to the product, indeed.

Reply:
It's included in `app/javascript/packs/Search.jsx` currently, which I think is just having the side effect that it's available throughout the app (since search is always in the header). We should bring it into `packs/base.jsx` though as it makes a lot more sense there

Reply:
Sooo I moved it to packs/base.jsx, in this PR. I hope you don't mind, Mr. Nick "Let's split it into smaller PRs" Taylor :D